tools/proofers/character_selector.js

```diff
@@ -18,6 +18,84 @@ $(function () {
         top.focusText();
     }
 
+    // The mru (most recently used) list stores a set of characters
+    // with time of last use and title.
+    // on page load get data and draw html.
+    // on each char: if already in the set update time
+    // else add it to start of array
+    // if overflowing find oldest and remove it
+    // redraw if changed
+    // on page unload, save data
+
+    var mru;
+    var storageKey = "mru_" + projectID;
+    var mruString = localStorage.getItem(storageKey);
+    if (mruString) {
+        mru = JSON.parse(mruString);
+    }
+    if(!Array.isArray(mru)) {
+        // this should only happen during development changes
+        mru = [];
+    }
+
+    window.addEventListener('beforeunload', function() {
+        localStorage.setItem(storageKey, JSON.stringify(mru));
+    });
+
+    const mruColumns = 12;
+    const mruMax = 2 * mruColumns;
+
+    function drawRow(mruRow) {
+        // it is not necessary to escape the character or title
+        var row = $('<div />').addClass('table-row');
+        mruRow.forEach(function(element) {
+            row.append($('<div />')
+                .addClass('table-cell')
+                .append($('<button />', {type: "button", title: element.title})
+                    .addClass('picker')
+                    .text(element.character)
+                )
+            );
+        });
+        return row;
+    }
+
+    function drawMru() {
+        // this duplicates the html defined in CharacterSelector.inc
+        var mruUpper = mru.slice(0, mruColumns);
+        var mruLower = mru.slice(mruColumns);
+        // this is the selector for mru code
+        $("._26237831463535303b").html(drawRow(mruUpper))
```

Comment:
What is this ID?

Reply:
It was bintohex of the clock entity code. The new change moves all the mru code into the js file so it is not needed now.